cmd/mount2/node.go

```diff
@@ -405,3 +406,27 @@ func (n *Node) Rename(ctx context.Context, oldName string, newParent fusefs.Inod
 }
 
 var _ = (fusefs.NodeRenamer)((*Node)(nil))
+
+var _ fusefs.NodeReadlinker = (*Node)(nil)
+
+// read symbolic link target
+func (n *Node) Readlink(ctx context.Context) (ret []byte, err syscall.Errno) {
```

Comment:
Readlink should likely be a vfs function I think, `vfs.Readlink` analagous to `os.Readlink`

Reply:
Done, though now symlink is created correctly (server wise) but local fs (fuse) thrown an error.
Any idea ??

```
 pasnox  ~  Projects  rclone  ./rclone mount storj-s3-pasnox:test /home/pasnox/public_html/storj-mount -v                                                                                                     SIGINT   master 
2022/07/29 20:00:40 INFO  : S3 bucket test: poll-interval is not supported by this remote
2022/07/29 20:00:49 INFO  : N3.rclonelink: Copied (new)
2022/07/29 20:00:49 fuse: panic in handler for Symlink [ID=0x6 Node=0x1 Uid=1000 Gid=1001 Pid=30309] from "N3" to target "Native-2": interface conversion: interface is nil, not fs.Node
goroutine 82 [running]:
bazil.org/fuse/fs.(*Server).serve.func2()
        /home/pasnox/go/pkg/mod/bazil.org/fuse@v0.0.0-20200524192727-fb710f7dfd05/fs/serve.go:994 +0x14e
panic({0x24cc400, 0xc000463560})
        /usr/lib/go/src/runtime/panic.go:838 +0x207
github.com/rclone/rclone/cmd/mount.(*Dir).Symlink(0xc0003320b0, {0x2c86860?, 0xc0003320b0?}, 0xc000c7e3c0)
        /home/pasnox/Projects/rclone/cmd/mount/dir.go:266 +0x3c5
bazil.org/fuse/fs.(*Server).handleRequest(0xc0007b8000, {0x2c98520, 0xc000a860c0}, {0x2c86860?, 0xc0003320b0}, 0xc0000644c0, {0x2c92a80?, 0xc000c7e3c0?}, 0xc0007bdf08)
        /home/pasnox/go/pkg/mod/bazil.org/fuse@v0.0.0-20200524192727-fb710f7dfd05/fs/serve.go:1101 +0x296a
bazil.org/fuse/fs.(*Server).serve(0xc0007b8000, {0x2c92a80, 0xc000c7e3c0})
        /home/pasnox/go/pkg/mod/bazil.org/fuse@v0.0.0-20200524192727-fb710f7dfd05/fs/serve.go:1015 +0x634
bazil.org/fuse/fs.(*Server).Serve.func1()
        /home/pasnox/go/pkg/mod/bazil.org/fuse@v0.0.0-20200524192727-fb710f7dfd05/fs/serve.go:512 +0x69
created by bazil.org/fuse/fs.(*Server).Serve
        /home/pasnox/go/pkg/mod/bazil.org/fuse@v0.0.0-20200524192727-fb710f7dfd05/fs/serve.go:510 +0x3be
```

Reply:
I found out I would need to still fill / create the fuse node.
How would I test mount2/cmount on linux ?
Right now i'm just stucked using mount.

Reply:
Nevermind.. find out myself.